main.go

```diff
@@ -16,6 +16,25 @@ import (
 	_ "github.com/heroku/x/hmetrics/onload"
 )
 
+var (
+	downloadConcurrency  int
+	maxPayloadSizeInByte int64
+)
+
+func init() {
+	downloadConcurrency, _ = strconv.Atoi(os.Getenv("DOWNLOAD_CONCURRENCY"))
+	if downloadConcurrency <= 0 {
+		downloadConcurrency = 10
+		log.Println("DOWNLOAD_CONCURRENCY not specified. Setting it to", downloadConcurrency)
+	}
+
+	maxPayloadSizeInByte, _ = strconv.ParseInt(os.Getenv("MAX_PAYLOAD_SIZE_IN_BYTES"), 10, 64)
+	if maxPayloadSizeInByte <= 0 {
+		maxPayloadSizeInByte = 32 * 1024 * 1024 * 1024
```

Comment:
@maknahar 
I think this is 32GB. Correct me If I'm wrong.

Reply:
yeah. It is.

Reply:
No... It is 32 GB...

Reply:
You guys are correct. Changing it.